src/Kovan-DssSpell.t.sol

```diff
@@ -1179,6 +1179,35 @@ contract DssSpellTest is DSTest, DSMath {
             calc_step:    0,
             calc_cut:     0
         });
+        afterSpell.collaterals["MATIC-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      10 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
+            line:         3 * MILLION, // TODO: is this correct? currently setting this according to dc-iam gap
```

Comment:
In the test `line` can be set to zero if the auto-line is in use, since the test uses `aL_line` value instead. You can leave `line` at 3m in the spell though.

Reply:
done